lib/wongi-engine/beta/ncc_node.rb

```diff
@@ -58,7 +58,7 @@ def ncc_deactivate token
       def refresh_child child
         tokens.each do |token|
           if token.ncc_results.empty?
-            child.beta_activate Token.new( child, t, nil, { } )
+            child.beta_activate Token.new( child, token, nil, { } )
```

Comment:
looks good to me, I think we should merge this pull request asap

Reply:
I think you'll find master already has this change.